lang/pt-br.xml

```diff
@@ -276,7 +276,7 @@ Não desligue o sistema ou feche o aplicativo.</warning_saving>
 
 	<game_data>
 		<app_close>O seguinte aplicativo será fechado.</app_close>
-		<data_delete>Dados a serem excludíos:</data_delete>
+		<data_delete>Dados a serem excluídos:</data_delete>
```

Comment:
revert

Reply:
why though? those translations are right afaik

Reply:
"excludios" isn't even a word in portuguese

Reply:
@Croden1999 you can check please
@pedrozzz0 here some translate is basée on officiel vita string

Reply:
> @Croden1999 you can check please @pedrozzz0 here some translate is based on official vita string

i check iit again, no any problem.